app/ui-message/client/blocks/MessageBlock.js

```diff
@@ -134,7 +133,7 @@ export const modalBlockWithContext = ({
 				<Modal open id={id} ref={ref}>
 					<Modal.Header>
 						{/* <Modal.Thumb url={`api/apps/${ context.appId }/icon`} /> */}
-						<Modal.Thumb url={thumb} />
+						<Modal.Thumb url={`/api/apps/icon/${ data.appId }`} />
```

Comment:
this is not the best way to get the full URL, pls advice

Reply:
there is a context to set the prefix path

Reply:
I have found: https://github.com/RocketChat/Rocket.Chat/blob/982ee13018129d5bb6d99cf2b8676a0fef3a3084/client/contexts/ServerContext.js#L12

but looks like I cannot use it because you're not using `ServerProvider` that implements it:
https://github.com/RocketChat/Rocket.Chat/blob/982ee13018129d5bb6d99cf2b8676a0fef3a3084/client/providers/ServerProvider.js#L42

what should I do?

Reply:
idk the advantages using a context, so I changed to use `getURL` that is widely used.